src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const status = person.sex === 'm'
+    ? 'is my wife'
+    : 'is my husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {
+      person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )
+      }
+      <p className="Person__partner">
+        { !person.partnerName
+          || person.partnerName === 'Some Partner'
+          ? 'I am not married'
+          : `${person.partnerName} ${status}`
```

Comment:
Better save it to the variable, also why do you need this weird `Some Partner` check?
```suggestion
        { !person.partnerName
          ? 'I am not married'
          : `${person.partnerName} ${status}`
```

Reply:
I have problem with test 
AssertionError: Timed out retrying after 4000ms: expected '<p.Person__partner>' to contain text 'not married', but the text was 'Some Partner is my husband' 

- 'Some Partner is my husband'
+ 'not married'

const tanya = {
  name: 'Tanya',
  sex: 'f',
  age: 28,
  isMarried: false,
  partnerName: 'Some Partner'
};